src/spicedb-common/validationfileformat.test.ts

```diff
@@ -1,21 +1,22 @@
 import { parseValidationYAML } from "./validationfileformat";
+import { describe, it, expect } from 'vitest'
 
 describe("parsing validation YAML", () => {
   it("returns undefined for an empty file", () => {
     expect(parseValidationYAML("")).toEqual({
-      message: "data must be object",
+      message: "data should be object",
```

Comment:
I'm guessing these tests stopped passing on a recent SpiceDB wasm upgrade, but we don't run the tests in CI so we missed it. I'm going to enable tests as a follow-on to this.